sim/midas/src/main/cc/bridges/fpga_managed_stream.h

```diff
@@ -6,7 +6,25 @@
 #include <functional>
 #include <string>
 
-#include "bridge_stream_driver.h"
+#include "managed_stream.h"
+
+class FPGAManagedStreamIO {
```

Comment:
The difficulty with this definition of the interface is that it assumes a particular organization of the circular buffers in memory -- i.e, that the are contiguously allocated. I think this is fine for now but it may need to change when i actually implement this for Vitis, such that the underlying platform is responsible for the allocation and instead returns a series of pointers/references to allocated buffers for the streams.

Reply:
Yes, that sounds like a good idea.